api/HTMLVideoElement.json

```diff
@@ -513,7 +513,7 @@
               "version_added": false
             },
             "ie": {
-              "version_added": "10"
+              "version_added": false
```

Comment:
This is `msIsStereo3D` and came up in another review as more complicated than it seems at first. Can you find that review and see what links we dug up? We should probably add some notes here to avoid a change like this slipping through.